data/akshaykamath45.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Akshay Kamath",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Learning  **Web Development**.Computer Engineering Student",
+  "avatar":"https://github.com/akshaykamath45.png",
```

Comment:
```suggestion
```

Reply:
I have directly used  `Commit suggestion`,is it fine